react/Autosuggest/Autosuggest.test.js

```diff
@@ -32,70 +20,54 @@ const getAutosuggestProps = (suggestions = []) => ({
 });
 
 describe('Autosuggest', () => {
-  let element, autosuggest, errors, suggestions, suggestionsContainer, input;
-
   beforeEach(() => {
-    errors = [];
-
-    sinon.stub(console, 'error', errorMessage => {
-      errors.push(errorMessage);
-    });
+    global.console = { error: jest.fn() };
   });
 
   afterEach(() => {
-    console.error.restore();
+    jest.clearAllMocks();
```

Comment:
It's still clobbering `global.console`, though. Jest won't be able to reverse this.